src/main/java/com/example/jwt/entity/User.java

```diff
@@ -18,6 +16,7 @@
 @Getter
 @Builder
 @Table(name = "users")
+@AllArgsConstructor
 public class User implements Serializable {
```

Comment:
this class was renamed to `AppUser`
what's happened?

Reply:
Nope, I have changed it back to user.